packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -1484,20 +1494,67 @@ export class IrisGrid extends Component<IrisGridProps, IrisGridState> {
       }
       const columnIndex = model.getColumnIndexByName(column.name);
       assertNotNull(columnIndex);
-      if (value === undefined) {
-        this.removeQuickFilter(columnIndex);
-      } else if (value === null) {
-        this.setQuickFilter(columnIndex, column.filter().isNull(), '=null');
+      filterList.sort((a, b) => {
```

Comment:
This whole function could be extracted into a utils function, add some unit testing around it.